docs/cli.md

```diff
@@ -178,6 +190,40 @@ This command requires that a wallet is connected to the CLI...
 
 ---
 
+**`nexus gas expiry enable --tool-fqn <fqn> --owner-cap <object_id> --cost-per-minute <mist>`**
```

Comment:
ditto re --tool-fqn

Reply:
Adding to https://github.com/Talus-Network/nexus-sdk/pull/162#discussion_r2084463210 here - this command is under `nexus gas` and not `nexus tool` so it's not even implied here that it should be a `tool-fqn`.